internal/pkg/term/syncbuffer/termprinter.go

```diff
@@ -144,34 +134,32 @@ func (ltp *LabeledTermPrinter) lastNLines(logs []string) []string {
 }
 
 // writeLines writes a label and output logs to the terminal associated with the TermPrinter.
-// Returns the number of lines needed to erase based on terminal width,
-// or an error when failed to fetch terminal width.
-func (ltp *LabeledTermPrinter) writeLines(label string, lines []string) (int, error) {
-	width, err := ltp.terminalWidth(ltp.term)
-	if err != nil {
-		return 0, fmt.Errorf("get terminal width: %w", err)
-	}
+// Returns the number of lines needed to erase based on terminal width.
+func (ltp *LabeledTermPrinter) writeLines(label string, lines []string) int {
 	var numLines float64
 	writeLine := func(line string) {
 		fmt.Fprintln(ltp.term, line)
 		if len(line) == 0 {
 			numLines++
 			return
 		}
-		numLines += math.Ceil(float64(len(line)) / float64(width))
+		numLines += math.Ceil(float64(len(line)) / float64(terminalWidth(ltp.term)))
 	}
 	writeLine(label)
 	for _, line := range lines {
 		writeLine(fmt.Sprintf("%s%s", strings.Repeat(" ", ltp.padding), line))
 	}
-	return int(numLines), nil
+	return int(numLines)
 }
 
-// terminalWidth returns the width of the terminal or an error if failed to get the width of terminal.
-func terminalWidth(fw FileWriter) (int, error) {
-	width, _, err := term.GetSize(int(fw.Fd()))
-	if err != nil {
-		return 0, err
+// terminalWidth returns the width of the terminal associated with the given FileWriter.
+// If the FileWriter is not associated with a terminal, it returns a default terminal width.
+func terminalWidth(fw FileWriter) int {
+	terminalWidth := defaultTerminalWidth
+	if term.IsTerminal(int(fw.Fd())) {
+		if width, _, err := term.GetSize(int(fw.Fd())); err == nil {
+			terminalWidth = width
+		}
```

Comment:
nit:
```suggestion
	if !term.IsTerminal(int(fw.Fd())) {
	  return defaultTerminalWidth
	}
        if width, _, err := term.GetSize(int(fw.Fd())); err == nil { // NOTE: swallow the error because ........
		return width
	}
	return defaultTerminalWidth
```